Source/Metrics/Metric.swift

```diff
@@ -69,38 +69,38 @@ struct Metric {
         self.isBackground = false
     }
     
-    static func incrementMetricWithName(name: String, category: MetricsCategory) -> Metric {
+    static func incrementMetricWithName(_ name: String, category: MetricsCategory) -> Metric {
```

Comment:
Should we update the function name to consist with naming conversation in other files?